app/views/devise/sessions/new.html.erb

```diff
@@ -1,14 +1,31 @@
-<h2>Log in</h2>
 
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RecipeApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+   <%= stylesheet_link_tag "login"%>
+    
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body class='uniq'>
```

Comment:
With Rails, we don't need to write these HTML tags. Rails view files will automatically include them. Kindly remove these lines.